rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -1353,6 +1385,76 @@ public ResponseEntity loadAssessmentSummaryInfo(
         return new ResponseEntity<>(assessmentSummaryMap, HttpStatus.OK);
     }
 
+    @Operation(
+            summary = "Check cyclic hierarchy of a release with other releases.",
+            description = "Check cyclic hierarchy of a release with other releases."
+    )
+    @RequestMapping(value = RELEASES_URL + "/{id}/checkCyclicLink", method = RequestMethod.POST)
+    public ResponseEntity<?> checkForCyclicReleaseLink(
+            @Parameter(description = "The ID of the checking release.")
+            @PathVariable("id") String releaseId,
+            @RequestBody Map<String, Set<String>> relationshipReleaseIds
```

Comment:
Please add example for this in OpenAPI docs. A sample can be:

```java
@Parameter(description = "Releases to check",
        schema = @Schema(example = """
                {
                  "linkedReleases": ["3765276512", "987456"],
                  "linkedToReleases": ["12121212"]
                }""")
)
```

Reply:
@GMishx thanks for your comment, I fixed it